src/index.html

```diff
@@ -6,9 +6,59 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+          href="https://fonts.gstatic.com"
+          crossorigin="anonymous">
```

Comment:
```suggestion
    <link
      rel="preconnect"
      href="https://fonts.gstatic.com"
      crossorigin="anonymous"
    >
```
Checklist: "Keep your attributes correctly formatted"
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/7cb8b294-999a-4b66-9675-b3d1a7b2dd27)